third_party/webcomponentsjs/ShadowCSS.js

```diff
@@ -121,152 +121,178 @@
   in comments in lieu of the next selector when running under polyfill.
 */
 
-export const ShadowCSS = {
-  strictStyling: false,
-  // change a selector like 'div' to 'name div'
-  /** @this {ShadowCSS} */
-  scopeRules: function(cssRules, scopeSelector, opt_transformer) {
-    var cssText = '';
-    if (cssRules) {
-      Array.prototype.forEach.call(cssRules, function(rule) {
-        if (rule.selectorText && (rule.style && rule.style.cssText !== undefined)) {
-          cssText += this.scopeSelector(rule.selectorText, scopeSelector,
-            this.strictStyling, opt_transformer) + ' {\n\t';
-          cssText += this.propertiesFromRule(rule) + '\n}\n\n';
-        } else if (rule.type === CSSRule.MEDIA_RULE) {
-          cssText += '@media ' + rule.media.mediaText + ' {\n';
-          cssText += this.scopeRules(rule.cssRules, scopeSelector);
-          cssText += '\n}\n\n';
-        } else {
-          // KEYFRAMES_RULE in IE throws when we query cssText
-          // when it contains a -webkit- property.
-          // if this happens, we fallback to constructing the rule
-          // from the CSSRuleSet
-          // https://connect.microsoft.com/IE/feedbackdetail/view/955703/accessing-csstext-of-a-keyframe-rule-that-contains-a-webkit-property-via-cssom-generates-exception
-          try {
-            if (rule.cssText) {
-              cssText += rule.cssText + '\n\n';
-            }
-          } catch(x) {
-            if (rule.type === CSSRule.KEYFRAMES_RULE && rule.cssRules) {
-              cssText += this.ieSafeCssTextFromKeyFrameRule(rule);
-            }
+// [@ampproject difference with original]
```

Comment:
I recommend disabling whitespace to read this diff. [Click here to disable](&w=1) or use the dropdown on ⚙️:

<img width="801" alt="image" src="https://user-images.githubusercontent.com/254946/161680776-d4334584-4ce0-42d1-85d9-79b5e45c69f8.png">